vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -701,20 +701,30 @@ def get_moe_configs(
     block_shape = [block_n, block_k] if block_n and block_k else None
     json_file_name = get_config_file_name(E, N, dtype, block_shape)
 
-    config_file_path = os.path.join(
+    config_file_paths = []
+
+    default_config_file_path = os.path.join(
         os.path.dirname(os.path.realpath(__file__)), "configs", json_file_name)
-    if os.path.exists(config_file_path):
-        with open(config_file_path) as f:
-            logger.info("Using configuration from %s for MoE layer.",
-                        config_file_path)
-            # If a configuration has been found, return it
-            return {int(key): val for key, val in json.load(f).items()}
+    config_file_paths.append(default_config_file_path)
+
+    user_defined_config_folder = os.getenv("VLLM_FUSED_MOE_CONFIG_FOLDER", None)
```

Comment:
define all env vars in https://github.com/vllm-project/vllm/blob/main/vllm/envs.py with comments